grammar/mongo.g4

```diff
@@ -73,7 +73,7 @@ BooleanLiteral: 'true' | 'false';
 NumericLiteral: '-'? DecimalLiteral;
 
 DecimalLiteral:
-	DecimalIntegerLiteral '.' DecimalDigit* ExponentPart?
+	DecimalIntegerLiteral '.' DecimalDigit+ ExponentPart?
```

Comment:
Why this change?

Reply:
> I realized while testing that on the way to typing out 1.324, "1." would give an error. 1. is not json parsable. So this change enables the squigglies to point at an error instead of the error popup.

okay